src/scripts/breeding/BreedingController.ts

```diff
@@ -212,6 +212,7 @@ class BreedingController {
             case 'timesHatched': return `Hatches: ${App.game.statistics.pokemonHatched[pokemonData.id]() || 0}`;
             case 'breedingEfficiency': return `Efficiency: ${((pokemon.baseAttack * (GameConstants.BREEDING_ATTACK_BONUS / 100) + pokemon.proteinsUsed()) / pokemonMap[pokemon.name].eggCycles).toLocaleString('en-US', { maximumSignificantDigits: 2 })}`;
             case 'stepsPerAttack': return `Steps/Att: ${(App.game.breeding.getSteps(pokemonMap[pokemon.name].eggCycles) / (pokemon.baseAttack * (GameConstants.BREEDING_ATTACK_BONUS / 100) + pokemon.proteinsUsed())).toLocaleString('en-US', { maximumSignificantDigits: 2 })}`;
+            case 'dexId': return `Pokedex ID: ${pokemon.id}`;
```

Comment:
Do you think this would look better as `#000` instead of `Pokedex ID: 0`?

Reply:
Idk, im not imaginative. i just try to make happen what people ask.

Reply:
No worries, I think maybe to make it more consistent with the pokédex we should.

Reply:
Will do.

Reply:
Updated, hopefully i did it right.